ros2/test.py.tpl

```diff
@@ -17,7 +17,7 @@ sys.argv = sys.argv[:1] + [
     LAUNCH_FILE,
 ] + sys.argv[1:]
 
-test_outputs_dir = os.environ.get('TEST_UNDECLARED_OUTPUTS_DIR')
+test_outputs_dir = os.environ.get('TEST_UNDECLARED_OUTPUTS_DIR') or os.environ.get('TEST_TMPDIR')
```

Comment:
https://google.github.io/styleguide/pyguide.html#2144-decision: get() can return None and one should check that as `if foo is None:`

Reply:
Thx, didn't know the Google Styleguide disallows `or` short-circuiting!